### DISCUSSION TITLE
usehead vs components for meta

### DISCUSSION THREAD

**Discussion by abhitheawesomecoder** [nuxt/nuxt#16355] [General]
I am not able to make out why we have usehead composable and Meta components both for same purpose. Please help.

**Comment by TheAlexLichter** [comment]
Both, the `useHead` composable and  meta components (`<Head>` + `<Title>` / `<Meta>` / ...) are ways to define meta tags in your app. 

Right now, `useHead` is more common to use though (as e.g. `vue-meta` did the same before and logic stays in script part).
